examples/jsm/renderers/webgpu/nodes/WGSLNodeBuilder.js

```diff
@@ -599,6 +601,44 @@ ${ flowData.code }
 
 	}
 
+	getDirective( name, shaderStage = this.shaderStage) {
+
+		const stage = this.directives[ shaderStage ] || ( this.directives[ shaderStage ] = [] );
+		stage.push(name);
+
+	}
+
+	getDirectives( shaderStage ) {
+
+		const snippets = [];
+		const directives = this.directives[ shaderStage ];
+
+		if ( directives !== undefined ) {
+
+			for ( const directive of directives ) {
+
+				snippets.push(`enable ${directive}`);
```

Comment:
It's be necessary apply mrdoob code style in the code for example: ```snippets.push( `enable ${ directive }` );```

Reply:
Fixed style issues in wgsl node builder